README.md

```diff
@@ -14,4 +14,17 @@
        git push -u origin main | git push --set-upstream origin main
      ```
      
-     
+  ## Creating  pull Requests
+    1. Create the new branch for the feature and commit it to the the repository you forked in your account
+      ```git
+       git checkout -b <feature_branch>
+
+       git push -u origin  <feature_branch>
+      ```
+    2. After committing the feature branch, a new pull request appears. You can now compare the differences between the the two branches, in this case the PULL-REQUESTS branch and the main branch of the repository you forked from. It uses the git diff command to compare the two branches. It shows  files have been added, removed, deleted, changed and so on.
+      ```git
+
+         git diff main PUll-REQUESTS 
+         
+      ````
+      
```

Comment:
I would like you describe the process better, then I merge the chnages

Reply:
working on it

Reply:
Made the appropriate changes, could you check it out.